contracts/validator-manager/tests/ValidatorManagerTests.t.sol

```diff
@@ -238,19 +229,15 @@ abstract contract ValidatorManagerTest is Test {
     // reference to the abstract type.
     function testResendRegisterValidatorMessage() public {
         bytes32 validationID = _setUpInitializeValidatorRegistration(
-            DEFAULT_NODE_ID,
-            DEFAULT_SUBNET_ID,
-            DEFAULT_WEIGHT,
-            DEFAULT_EXPIRY,
-            DEFAULT_BLS_PUBLIC_KEY
+            DEFAULT_NODE_ID, DEFAULT_SUBNET_ID, DEFAULT_WEIGHT, DEFAULT_BLS_PUBLIC_KEY
         );
         (, bytes memory registerL1ValidatorMessage) = ValidatorMessages
             .packRegisterL1ValidatorMessage(
             ValidatorMessages.ValidationPeriod({
                 subnetID: DEFAULT_SUBNET_ID,
                 nodeID: DEFAULT_NODE_ID,
                 blsPublicKey: DEFAULT_BLS_PUBLIC_KEY,
-                registrationExpiry: DEFAULT_EXPIRY,
+                registrationExpiry: uint64(block.timestamp) + 1 days,
```

Comment:
It looks like `DEFAULT_EXPIRY` is now gone and can be deleted from the constants above but can also make 1 days a new `EXPIRY_LENGTH` constant which would make line 421 more self-explanatory